alerts/build.gradle.kts

```diff
@@ -31,6 +30,7 @@ kotlin {
         getByName("commonMain") {
             dependencies {
                 val ext = (gradle as ExtensionAware).extra
+                implementation(project(":architecture", ""))
```

Comment:
Not sure that I fully understand the moving of this dependency (but I need to read more)
Reading further in this review, I can see that you are now expecting `LifecycleSubscribable` in common code.
I've yet to fully understand why that was needed, but I can see that the Android have a meaningful `actual` for it while other platforms don't.

This makes me wonder: wasn't it correct before? Or doesn't it need to be added at least to AndroidLibMain dependencies?

Reply:
I had to move it because Android uses reflection to determine whether a Viewmodel object iss Lifecycle aware. If we're now going to pass the common BaseAlertsPresenter.Builder to the viewModel, that reflection no longer works, since its specified in the subclass. Therefore I simplified things by making the interface common, but having it as an empty interface on non-Android platform.

Reply:
Clear, as I was reading through I understood why you made the change.
As I said, my concern was about the position of the dependency but you pointed out that having it in common is enough